discord/client.py

```diff
@@ -464,6 +464,24 @@ def login(self, email, password):
             passing status code.
         """
 
+        if email == "token":
+            log.info('logging in using static token')
+            self.token = password
+            self.headers['authorization'] = 'Bot {}'.format(self.token)
+            resp = yield from self.session.get(endpoints.ME, headers=self.headers)
+            log.debug(request_logging_format.format(method='GET', response=resp))
+
+            if resp.status != 200:
+                yield from resp.release()
```

Comment:
You never release the response if it is successful - my debug log has been complaining about this. :)